string   ModuleName
duration PauseTime
duration ExecutionTime